kore/src/Kore/Reachability/Claim.hs

```diff
@@ -334,9 +334,11 @@ transitionRule claims axiomGroups = transitionRuleWorker
             >>= return . applyResultToClaimState
```

Comment:
We probably want to do the same here, for claims. Can you give this a try as well?